MSAL/test/unit/ios/unit-test-host/AppDelegate.h

```diff
@@ -0,0 +1,36 @@
+//------------------------------------------------------------------------------
```

Comment:
"Add a clean unit test host app, as changes in init order are now exposing test failures (and maybe were before that we just didn't see...)"